test/components/segmented-control/segmented-control.test.js

```diff
@@ -42,6 +42,21 @@ describe('<SegmentedControl />', () => {
     expect(spy.calledOnce).to.equal(expected);
   });
 
+  it('should call user supplied onChange when an input with type checkbox is changed to checked', () => {
+    const spy = sinon.spy();
+    const wrapper = renderComponent({ onChange: spy, type: 'checkbox', children: [<span value={1}>1</span>, <span value={2}>2</span>] });
+    const actual = wrapper.find('input[type="checkbox"][value=2]');
+    actual.simulate('change', { target: { checked: true }, currentTarget: { checked: true } });
+    expect(spy.calledOnce).to.equal(true);
+  });
+
+  it('should set state to true when an input with type radio is changed to checked', () => {
+    const wrapper = renderComponent({ type: 'checkbox', children: [<span value={1}>1</span>, <span value={2}>2</span>] });
```

Comment:
Did you intend for this to be `type: 'checkbox'` or `type: 'radio'`? Test description would indicate the latter.

Reply:
Test description was wrong, now it´s updated.